modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.util.regex.txt

```diff
@@ -29,5 +29,31 @@ class Pattern -> java.util.regex.Pattern extends Object {
 }
 
 class Matcher -> java.util.regex.Matcher extends Object {
+  int end()
+  int end(int)
+  boolean find()
+  boolean find(int)
+  String group()
+  String group(int)
+  # String group(String)  How do I get this working?
```

Comment:
That is probably ok here but seems a bit breaky given that java allows it and we're exposing java APIs. We can just say that we don't support named groups and that'd be OK here but I have a suspicion something else will come up.